### ISSUE TITLE
5.2.2 - CrudSaga does not appear to be picking up actions

### ISSUE THREAD

**Issue by tryangul** [uniqueway/redux-crud-store#52]
Fetch action fires, but afterwards nothing happens. I see no request in my network tab, and no subsequent actions fire. I am struggling to debug given the lib is transpiled.

redux 3.7
redux-crud-store 5.2.2
redux-saga 0.15.3

Downgrading to 5.2.1 seems to fix it, however.

My Code:

dispatched action
```
meta: Object,
type: 'redux-crud-store/crud/FETCH',
payload: {
  method: 'get',
  path: '/materials',
  ...
}
```

store.js

```
/* global __DEV__ */
import { createStore, applyMiddleware, compose } from 'redux';
import createSagaMiddleware from 'redux-saga';
import { routerMiddleware } from 'react-router-redux';
import { createLogger } from 'redux-logger';
import { crudSaga, ApiClient } from 'redux-crud-store'
import createHistory from 'history/createBrowserHistory';
import rootReducer from './reducer';

const client = new ApiClient({ basePath: 'http://localhost:8080' });
const crudMiddleware = createSagaMiddleware();

const history = createHistory();
const initialState = {};
const enhancers = [];

const middleware = [
  routerMiddleware(history),
  crudMiddleware
];

// add dev logging/dev tools in dev
if (__DEV__) {
  const __REDUX_DEVTOOLS_EXTENSION__ = window.__REDUX_DEVTOOLS_EXTENSION__;

  if (typeof devToolsExtension === 'function') {
    enhancers.push(__REDUX_DEVTOOLS_EXTENSION__())
  }

  const logger = createLogger({
    stateTransformer: state => ({ ...state, models: state.models.toJSON() })
  });

  middleware.push(logger)
}

const composedEnhancers = compose(
  applyMiddleware(...middleware),
  ...enhancers
);

const store = createStore(
  rootReducer,
  initialState,
  composedEnhancers
);

crudMiddleware.run(crudSaga(client));

export {
  store,
  history,
}
```

component
```
import React, { PureComponent } from 'react';
import { connect } from 'react-redux';

import { select } from 'redux-crud-store'

import { fetchMaterials } from '../../redux/modules/materials'

import './MyComponent.css';

class MyComponent extends PureComponent {
  componentWillMount() {
    const { materials, dispatch } = this.props

    if (materials.needsFetch) {
      dispatch(materials.fetch);
    }
  }

  render() {
    return (...jsx stuff)

  }
}

function mapStateToProps(state, ownProps) {
  return { materials: select(fetchMaterials(), state.models) };
}

export default connect(mapStateToProps)(DataTable)

```

action
```
const MODEL = 'materials';
const PATH = '/materials';

export function fetchMaterials(params = {}) {
  return fetchCollection(MODEL, PATH, params);
}

```

**Comment by devvmh** [comment]
Ah! Excellent. Thanks a lot for reporting this. I experienced this too, and this report is super helpful.  I'll aim to push a fix for this this week, provided I get time. I  the meantime, definitely switch back to 5.2.1.

**Comment by tryangul** [comment]
No problem. Now that I have it working, I am loving the library!

**Comment by devvmh** [comment]
Glad to hear it! The goal of the library is to save you time, so if you find anything repetitive or have other feature requests definitely let me know and I'll see if we can add those things in a future release!

Also, if any aspect was confusing to set up let me know and I'll see if we can rework the docs.

**Comment by devvmh** [comment]
Thanks again for reporting! This is fixed (at least on my machine) as of [v5.2.3](https://github.com/devvmh/redux-crud-store/releases/tag/v5.2.3)

Please test it out and re-open this issue if it still doesn't work.
